core/log-core.js

```diff
@@ -908,7 +908,7 @@ var bindDynLogLevel = function (token, logger) {
 
 // Verifies the given JWT and returns its payload.
 var verifyAndDecodeJWT = function (token, key) {
-    if (key == null || !token) {
+    if (key == null || typeof key !== "string" || !token) {
```

Comment:
why do we use 3 different comparison versions here?
looks a little bit convoluted, !key should also work right?
```suggestion
    if (!token || !key || typeof key !== "string") {
```

Reply:
Valid point 👍